proto/buf/alpha/registry/v1alpha1/repository.proto

```diff
@@ -70,6 +74,20 @@ service RepositoryService {
   rpc DeleteRepository(DeleteRepositoryRequest) returns (DeleteRepositoryResponse);
   // DeleteRepositoryByFullName deletes a repository by full name.
   rpc DeleteRepositoryByFullName(DeleteRepositoryByFullNameRequest) returns (DeleteRepositoryByFullNameResponse);
+  // DeprecateRepositoryByName deprecates the repository.
+  rpc DeprecateRepositoryByName(DeprecateRepositoryByNameRequest) returns(DeprecateRepositoryByNameResponse);
+  // UndeprecateRepositoryByName makes the repository not deprecated and removes any deprecation_message.
+  rpc UndeprecateRepositoryByName(UndeprecateRepositoryByNameRequest) returns(UndeprecateRepositoryByNameResponse);
+  // GetRepositoriesByFullName gets repositories by full name.
+  rpc GetRepositoriesByFullName(GetRepositoriesByFullNameRequest) returns (GetRepositoriesByFullNameResponse);
```

Comment:
Following the previous conventions, should this be `ListRepositoriesByFullName`? I know that it's not a paginated list, so it's not quite the same as the other `List` endpoints, but it's also getting multiple repository entities? Unclear, I leave this up to the jury.

Reply:
I think of a List* endpoint as one that returns all items that match a criteria vs this which returns a list with a 1:1 correlation of the input list.

Reply:
I think in this context, `Get` is right - I had the same instinct as @doriable but after reviewing other examples in our APIs, this makes sense to me